server/app/views/applicant/IneligibleTemplate.html

```diff
@@ -52,7 +52,8 @@
     </div>
     <div
       th:replace="~{applicant/ModalFragment:: modalContainer(<!--/* showProgramIndexLogin= */-->false,
-                                                             <!--/* showReviewLogin= */-->false)}"
+                                                             <!--/* showReviewLogin= */-->false,
```

Comment:
If all params are false, why is this needed?

Reply:
we still show the devtools modal in all pages